### PR TITLE
Small additions

### PR DESCRIPTION
**Pull request by Gamerein**
I fixed 2 server invite links that were outdated,
Added Discord Bot lists although maybe you should put those in the Bots section
Added the permissions calculator

Also, if you want you can include [this](https://github.com/Gamerein/rooCollection) collection of roo emojis to the Emojis section but I decided not to include it in the pr

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -125,6 +127,10 @@
 - [DiscordServ](https://discordserv.com/)
 - [Disboard](https://disboard.org/)
 
+### Bot lists:
+- [discordbots.org](https://discordbots.org/) | [server](https://discord.gg/EYHTgJX)
+- [bots.discord.pw](https://bots.discord.pw/) | [server](https://discord.gg/0cDvIgU2voWn4BaD)
+
```
Comment (byparallax) [inline readme.md]: Would it be possible to remove the servers, please?

**Comment by byparallax** [comment]
Regarding the Roos collection, since some of them are already listed [on this repository](https://github.com/sn0w/awesome-discord-emojis), I believe it would be better to send a PR to sn0w.

File: readme.md (hunk 1 above)
Comment (Gamerein) [reply readme.md]: done ed3ebed

**Comment by Gamerein** [comment]
I removed the links to the server of the discord bot lists. Is the PR ok now?

**Comment by byparallax** [comment]
**Hey @Gamerein!** 🎉

We really appreciate your contribution to the [Awesome Discord](https://github.com/jacc/awesome-discord) project!

If you own or maintain the project[s] added in your pull request, you should consider adding this wonderful badge to your github repository or website. Thanks!

```markdown
[![Mentioned in Awesome Discord](https://awesome.re/mentioned-badge.svg)](https://github.com/jacc/awesome-discord)
```
<p align='right'><a href="https://github.com/jacc/awesome-discord"></p></a>

If you're interested in the development of this awesome list, you should [join our discord server](https://awesome.discord.io/join).

**-byParallax and the Awesome Discord team**　:octocat:

**Pull request merged**